js/tc39/breaking_test_errors.json

```diff
@@ -4724,16 +4643,15 @@
   "test/language/statements/for-in/head-const-bound-names-dup.js-strict:true": "test/language/statements/for-in/head-const-bound-names-dup.js: unexpected error type (TypeError), expected (SyntaxError)",
   "test/language/statements/for-in/head-const-bound-names-fordecl-tdz.js-strict:true": "test/language/statements/for-in/head-const-bound-names-fordecl-tdz.js: Test262Error: Expected a ReferenceError to be thrown but no exception was thrown at all at harness/sta.js:22:9(49)",
   "test/language/statements/for-in/head-const-bound-names-in-stmt.js-strict:true": "test/language/statements/for-in/head-const-bound-names-in-stmt.js: error is not an object (Test262: This statement should not be evaluated.)",
+  "test/language/statements/for-in/head-let-bound-names-dup.js-strict:true": "test/language/statements/for-in/head-let-bound-names-dup.js: unexpected error type (TypeError), expected (SyntaxError)",
```

Comment:
This actually now returns the "correcter":
```
ERRO[0000] TypeError: file:///home/mstoykov/work/k6_copy/js/tc39/d.js: Duplicate declaration "x"
> 1 | for (let [x, x] in {}) {}
    |              ^
  2 |  at <eval>:2:28542(114)
```
instead of 
```
SyntaxError: Unexpected strict mode reserved word at file:///home/mstoykov/work/k6_copy/js/tc39/d.js:1:6
```
The error comes from babel transformation so :shrug: